website/src/components/breadcrumbs.tsx

```diff
@@ -17,7 +20,7 @@ const build = (
 ) => {
   const name = parts[index] === '' ? 'home' : parts[index];
   if (index === parts.length - ONE) {
-    nav.push(<li>
+    nav.push(<li key={name}>
```

Comment:
why does the li get a key property?

Reply:
I did it to deal with this React warning as described above

![image](https://github.com/Idrinth/api-bench/assets/99273340/0090f40f-c8d5-4458-91ae-ce3c21da1395)

If the warning is not that important I'll gladly remove it